Tests/Helpers/AbstractDatabaseTestCase.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Entity\Tests\Helpers;
+
+use Joomla\Database\DatabaseDriver;
+use PHPUnit\DbUnit\Database\DefaultConnection;
+use PHPUnit\DbUnit\DataSet\CsvDataSet;
+use PHPUnit\DbUnit\Operation\Composite;
+use PHPUnit\DbUnit\Operation\Factory;
+use PHPUnit\DbUnit\Operation\Operation;
+use PHPUnit\DbUnit\TestCase;
+
+/**
+ * Base test case for the database package
+ */
+abstract class AbstractDatabaseTestCase extends TestCase
+{
+	/**
+	 * The active database driver being used for the tests.
+	 *
+	 * @var  DatabaseDriver
+	 */
+	protected static $driver;
+
+	/**
+	 * The database driver options for the connection.
+	 *
+	 * @var  array
+	 */
+	protected static $options = array ();
+
+	/**
+	 * Sets up the fixture.
+	 *
+	 * This method is called before a test is executed.
+	 *
+	 * @return  void
+	 */
+	protected function setUp()
+	{
+		if (!static::$driver)
+		{
+			$this->markTestSkipped('There is no database driver.');
+		}
+
+		parent::setUp();
+	}
+
+	/**
+	 * This method is called after the last test of this test class is run.
+	 *
+	 * @return  void
+	 */
+	public static function tearDownAfterClass()
+	{
+		if (static::$driver !== null)
+		{
+			static::$driver->disconnect();
+			static::$driver = null;
+		}
+	}
+
+	/**
+	 * Gets the data set to be loaded into the database during setup
+	 *
+	 * @return CsvDataSet
+	 */
+	protected function getDataSet()
```

Comment:
This method should be inside your test class (`Tests/Models/User.php`) so you can have multiple sets of test data if you need it)